### ISSUE TITLE
Report diagnostic for passing a literal string without parameters as interpolated string argument

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#982]
There's no analyzer rule (anywhere, that I know of) that reports a diagnostic when a string literal without parameters is passed as value for an argument that is an interpolated string handler.

```c#
var x = string.Create(CultureInfo.InvariantCulture, $"string without parameters."); // NOK
var x = string.Create(CultureInfo.InvariantCulture, $"Current time is {DateTime.Now:D}."); // OK
```
